diff -Nur diacanvas2-0.10.0.orig/configure.in diacanvas2-0.10.0/configure.in
--- diacanvas2-0.10.0.orig/configure.in	2003-10-14 08:50:04.000000000 +0200
+++ diacanvas2-0.10.0/configure.in	2003-11-04 13:46:10.000000000 +0100
@@ -247,25 +247,6 @@
 AC_SUBST(PYGTK_DEFSDIR)
 AC_MSG_RESULT($PYGTK_DEFSDIR)
 
-AC_MSG_CHECKING(for gnome-python (for GnomeCanvas))
-prog="
-import sys
-import imp
-try:
-	import pygtk
-	pygtk.require('2.0')
-	#imp.find_module('gnome/canvas')
-	import gnome.canvas
-except ImportError:
-	sys.exit(1)
-sys.exit(0)"
-if $PYTHON -c "$prog" 1>&AC_FD_CC 2>&AC_FD_CC; then
-    AC_MSG_RESULT(yes)
-else
-    AC_MSG_RESULT(no)
-    enable_python=no
-fi
-
 fi # enable_python
 
 AM_CONDITIONAL(ENABLE_PYTHON, test "x$enable_python" = "xyes")
